cranelift/filetests/filetests/isa/x64/pinned-reg.clif

```diff
@@ -28,8 +30,14 @@ block0:
 
 ;   pushq   %rbp
 ;   movq    %rsp, %rbp
+;   subq    %rsp, $16, %rsp
+;   movq    %rsi, 0(%rsp)
```

Comment:
It's not clear to me why this test changed to use `%rsi` for the temporary, but that's what causes the new stack space.

Reply:
Scratch that -- it's the first preferred register, and also a clobber with sysv64.